python/GafferUI/SpreadsheetUI/_ClipboardAlgo.py

```diff
@@ -71,6 +71,9 @@ def rowData( row ) :
 	return True
 
 ## Builds a 'paste-able' data for the supplied plug matrix
+# For Spreadsheet rows, the matrix should consist of a single column
```

Comment:
This is pleasingly simple! Good to see that it fits into your adaptor scheme easily too.